README.md

```diff
@@ -53,7 +53,7 @@ Support this project by becoming a Solidus Ambassador. Your logo will show up he
 
 ## Summary
 
-Solidus is a complete open source ecommerce solution built with Ruby on Rails.
+Solidus is a complete open source e-commerce solution built with Ruby on Rails.
```

Comment:
I couldn't care less about which we go with, but for what it's worth, Shopify spells it "ecommerce", so that's what I do.